muxers/mplex/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-mplex"
 edition.workspace = true
 rust-version = { workspace = true }
 description = "Mplex multiplexing protocol for libp2p"
-version = "0.43.1"
+version = "0.44.0"
```

Comment:
No need for version bumps or changelog entries if the change only concern the tests (here and in all other affected packages) :)